hedera-mirror-web3/src/test/java/com/hedera/mirror/web3/evm/store/accessor/AccountDatabaseAccessorTest.java

```diff
@@ -59,6 +61,7 @@ class AccountDatabaseAccessorTest {
     private static final Optional<Long> timestamp = Optional.of(1234L);
     private static final int POSITIVE_BALANCES = 7;
     private static final int NEGATIVE_BALANCES = 8;
+    private static final long TREASURY_ACCOUNT_ID = SystemEntity.TREASURY_ACCOUNT.getNum();
```

Comment:
Change to store `EntityId`. Consider also moving to centralized spot in tests to stay DRY.
```suggestion
    private static final EntityId TREASURY_ACCOUNT_ID = SystemEntity.TREASURY_ACCOUNT.getScopedEntityId(CommonProperties.getInstance());
```